build/xwalk.gni

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) 2016 Intel Corporation. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# This file contains Crosswalk-feature-related build flags. Users should
+# import it into their build folder by run:
+#   echo 'import("//xwalk/build/xwalk.gni")' > out/your_folder/args.gn'
+# Then run:
+#   gn gen out/your_folder
+
+# Set it false when building content_shell.
+declare_args() {
+  is_build_crosswalk = true
+}
+
+if (is_build_crosswalk) {
+  target_os = "linux"
```

Comment:
This 3-line block does not make sense. The `target_*` variables should be set by GN and the toolchain code only, there is no need to change them here. As for `is_debug`, I don't see a reason to change the default for Crosswalk. If upstream enables debug builds by default, so should we.